### PR TITLE
Fix part #530: topic play dashed line issue fix in pre lollipop devices 

### PR DESCRIPTION
**Pull request by nikitamarysolomanpvt**
## Explanation

Topic play dashed line issue fix in pre lollipop devices- tested in 4.4.2 (Kitkat)

## Before 

Untitled

- [x] Dashed lines

## Current UI

small

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by nikitamarysolomanpvt** [comment]
@rajat your dev log issue mentioned below is addressed 

> **21st January**
> Topic play dashed line issue (Dashed line becomes single line in kitkat) -> No solution found and also, there is no trace of any such error on stackoverflow also.

File: app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt [hunk 1]
```diff
@@ -106,7 +107,7 @@ class StorySummaryAdapter(
         storySummaryViewModel.storySummary.chapterCount,
         completedChapterCount
       )
-
+      binding.topicPlayStoryDashedLineView.setLayerType(View.LAYER_TYPE_SOFTWARE, null)
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt]: Nit update: `binding.topicPlayStoryDashedLineView.setLayerType(View.LAYER_TYPE_SOFTWARE, /* paint= */ null)`

**Review by rt4914 — APPROVED** [review]
LGTM, one nit change suggested.

File: app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt]: Done.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Pull request merged**